### PR TITLE
Add image examples + cleanup failing asset downloads

### PR DESCRIPTION
**Pull request by wkozyra95**
20230831_10h30m33s_grim

gif, jpeg, png and svg example

### CODE REVIEW

File: compositor_pipeline/src/error.rs [hunk 1]
```diff
@@ -130,7 +130,7 @@ impl From<SceneUpdateError> for PipelineError {
         PipelineError {
             error_code: "UNKNOWN_SCENE_UPDATE_ERROR",
             error_type: ErrorType::ServerError,
-            message: err.to_string(),
+            message: format!("{:?}", err),
```
Comment (wkozyra95) [inline compositor_pipeline/src/error.rs:133]: temporary, hard to debug api errors in a meantime

File: examples/image.rs [hunk 2]
```diff
@@ -71,46 +71,132 @@ fn start_example_client_code() -> Result<()> {
...
+                "background_color_rgba": "#00000000",
+            },
+            {
+                "node_id": "svg_1_rescaled",
+                "type": "built-in",
+                "transformation": "transform_to_resolution",
+                "strategy": "fit",
+                "resolution": { "width": 960, "height": 540 },
+                "input_pads": ["svg_1"],
+                "background_color_rgba": "#00000000",
```
Comment (WojciechBarczynski) [inline examples/image.rs:167]: You don't have to specify this. It's default value

Comment (wkozyra95) [reply examples/image.rs:167]: > Received malformed request:\nmissing field `background_color_rgba`"

**Review by WojciechBarczynski — APPROVED** [review]
Rust Foundation is coming for you :P

File: examples/image.rs (hunk 2 above)
Comment (WojciechBarczynski) [reply examples/image.rs:167]: LOL I will fix it

**Pull request merged**